packages/grafana-ui/src/themes/_variables.scss.tmpl.ts

```diff
@@ -144,8 +144,8 @@ $input-border-radius-sm: 0 $border-radius-sm $border-radius-sm 0 !default;
 $label-border-radius: $border-radius 0 0 $border-radius !default;
 $label-border-radius-sm: $border-radius-sm 0 0 $border-radius-sm !default;
 
-$input-padding: ${theme.spacing.sm};
-$input-height: 35px !default;
+$input-padding: 0 ${theme.spacing.sm};
+$input-height: 36px !default;
```

Comment:
This seems like a variable that needs to exist in code theme as well and needs to be synced with the button heights.